### PR TITLE
Fix #251: pkcli.message

### LINKED ISSUE #251: add pkcli.message for output that is not an error but a warning or info
This would be similar to command_error without the exception. The message would go to stderr.

### PR DESCRIPTION
**Pull request by gurhar1133**
pkcli.message writes to stderr without raising
includes a test

### CODE REVIEW

File: pykern/pkcli/__init__.py [hunk 1]
```diff
@@ -72,6 +72,15 @@ def command_error(fmt, *args, **kwargs):
     raise CommandError(fmt.format(*args, **kwargs))
 
 
+def message(fmt, *args, **kwargs):
```
Comment (robnagler) [inline pykern/pkcli/__init__.py:75]: `command_info` is a better name, since it relates to `command_error`.

File: tests/pkcli_data/m1.in/x.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import pykern.pkcli
+import sys
+
+pykern.pkcli.message("this is a message")
```
Comment (robnagler) [inline tests/pkcli_data/m1.in/x.py:4]: This is not the intended usage. It should only be called from a pkcli. Better to structure it that way so people don't use it as a example (tests as documentation).

File: tests/pkcli_data/m1.in/x.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import pykern.pkcli
```
Comment (robnagler) [inline tests/pkcli_data/m1.in/x.py:1]: Not sure I understand the naming of this. We had p1, p2, and now m1.in. Perhaps just 1.in? The m1 makes me ask "what does m stand for"?

File: tests/pkcli_test.py [hunk 4]
```diff
@@ -64,6 +64,17 @@ def test_main3():
...
 
 
+def test_message():
+    from pykern import pksubprocess
+    from pykern.pkunit import case_dirs
+
+    for d in case_dirs("m1"):
+        pksubprocess.check_call_with_signals(
+            ["python", d.join("x.py")],
+            output="res.txt",
```
Comment (robnagler) [inline tests/pkcli_test.py:74]: res.txt maybe better x.out?

**Review by robnagler — CHANGES_REQUESTED** [review]

**Comment by gurhar1133** [comment]
made changes @robnagler

**Review by robnagler — APPROVED** [review]

**Pull request merged**